lib/node_modules/@stdlib/lapack/base/dlae2/docs/repl.txt

```diff
@@ -0,0 +1,76 @@
+
+{{alias}}( A, B, C, Out )
+    Computes the eigenvalues of a 2x2 symmetric matrix.
+
+    Indexing is relative to the first index. To introduce an offset, use typed
+    array views.
+
+    Parameters
+    ----------
+    A: number
+        The (0,0) element of matrix.
```

Comment:
```suggestion
        The (0,0) element of a 2x2 symmetric matrix.
```

"of matrix" is not grammatically correct. This should be fixed here and elsewhere.